etc/notes/CHANGES_4.0.0.md

```diff
@@ -324,6 +324,26 @@ const md5 = await new Promise((resolve, reject) => {
 await db.collection('fs.files').updateOne({ _id }, { $set: { md5 } });
 ```
 
+### BSON
+
+> Updated 4/19/2022
```

Comment:
nit: this date format is only valid mostly in the US and its territories. I'd change dates to explicitly call out the month so it's clear. Eg: Apr 4 2022

Reply:
Not something I've ever really thought about but good catch! I'll keep this in mind in the future too